rfc6962-bis.xml

```diff
@@ -1102,72 +1171,65 @@ entry specified by <spanx style="verb">start</spanx>.
           </t>
           <t>
-            An X509v3 certificate extension (see <xref target="cert_sctlist_extension"/>). This mechanism allows the use of unmodified TLS servers, but the SCTs cannot be updated on the fly. Since the logs that signed the SCTs won't necessarily be accepted by TLS clients for the full lifetime of the certificate, there is a risk that TLS clients will subsequently consider the certificate to be non-compliant and in need of re-issuance.
+            An X509v3 certificate extension (see <xref target="cert_transinfo_extension"/>). This mechanism allows the use of unmodified TLS servers, but the SCTs and inclusion proofs cannot be updated on the fly. Since the logs from where the SCTs and inclusion proofs originated won't necessarily be accepted by TLS clients for the full lifetime of the certificate, there is a risk that TLS clients will subsequently consider the certificate to be non-compliant and in need of re-issuance.
           </t>
         </list>
       </t>
       <t>
-        TLS servers SHOULD send SCTs from multiple logs in case one or more logs are not acceptable to the TLS client (for example, if a log has been struck off for misbehavior, has had a key compromise or is not known to the TLS client).
+        TLS servers SHOULD send SCTs or inclusion proofs from multiple logs in case one or more logs are not acceptable to the TLS client (for example, if a log has been struck off for misbehavior, has had a key compromise, or is not known to the TLS client).
       </t>
       <figure>
         <preamble>
-          Multiple SCTs are combined into an SCT list as follows:
+          Multiple SCTs, inclusion proofs, and indeed <spanx style="verb">TransItem</spanx> structures of any type, are combined into a list as follows:
         </preamble>
         <artwork>
-    opaque SerializedSCT&lt;1..2^16-1&gt;;
+    opaque SerializedTransItem&lt;1..2^16-1&gt;;
 
     struct {
-        SerializedSCT sct_list&lt;1..2^16-1&gt;;
-    } SignedCertificateTimestampList;</artwork>
+        SerializedTransItem trans_item_list&lt;1..2^16-1&gt;;
+    } TransItemList;</artwork>
       </figure>
       <t>
-        Here, <spanx style="verb">SerializedSCT</spanx> is an opaque byte string that contains the serialized SCT structure. This encoding ensures that TLS clients can decode each SCT individually (i.e., if there is a version upgrade, out-of-date clients can still parse old SCTs while skipping over new SCTs whose versions they don't understand).
+        Here, <spanx style="verb">SerializedTransItem</spanx> is an opaque byte string that contains the serialized <spanx style="verb">TransItem</spanx> structure. This encoding ensures that TLS clients can decode each <spanx style="verb">TransItem</spanx> individually (so, for example, if there is a version upgrade, out-of-date clients can still parse old SCTs while skipping over new SCTs whose versions they don't understand).
```

Comment:
Nit: Still mentioning SCTs while the rest of the sentence refers to SerializedTransItems.